### ISSUE TITLE
Feature request - Save settings

### ISSUE THREAD

**Issue by urbster1** [Eiyeron/lsdpatch#26]
Just had this thought - It would be really cool to save settings, so that you could easily load another ROM, load a settings file, and apply all kits, palettes, and fonts at once

**Comment by Eiyeron** [comment]
Yeah, I was thinking of that earlier, I wonder how I would do the thing.

1. Making a big file containing all kits/palettes/fotns
2. Referencing to other files.

**Comment by Eiyeron** [comment]
So, after a discussion with Defense Mechanism, here's the idea I got: packing edits into a zipfile. In said archive would be a list of palettes/fonts/kits that would be automatically slapped on a ROM and would call it a day.

The only thing that holds me for now to add such operation : I need to finish the rest. Sorry.

**Comment by Eiyeron** [comment]
Possible feature I just thought off : quick patching by drag'n'dropping both a pack and a ROM.

**Comment by urbster1** [comment]
what if the rom is also included in the zip! just patch everything into the included rom

**Comment by Eiyeron** [comment]
To keep track of which version the pack is designed for?

**Comment by urbster1** [comment]
not necessarily, just that it would require fewer arguments for the user if the rom was also in the zip. so if they get the zip they can add their rom to the zip, open zip with lsdpatch, lsdpatch spits out the patched rom. if no rom is included in the zip, prompt user for the rom first. something like that maybe?
